### ISSUE TITLE
[Tool bar] Inverted fill ghost variant has clipped focus indicator

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3169]
## Current behavior 

In [Tool bar Examples story](https://storybook.sanomalearning.design/?path=/story/actions-tool-bar--examples) for component presented on screenshot focus indicator is a white rectangle that partially is displayed on white background. Part of focus that is displayed on background is not visible due to very poor contrast. 
Focus indicator is the hardest to see for the first and last elements.

<img width="365" height="71" alt="Image" src="https://github.com/user-attachments/assets/2aa058c3-d0b2-41de-b40e-937c925a46d8" />

https://github.com/user-attachments/assets/98f18039-0414-4184-bea9-5102f2253079

## Suggested solution

It's recommended to change color of focus indicator for that component to one that has contrast ratio greater than 3:1 with blue background and white background. Example of such color can be #2B2B2B.
Other idea is to use two-color focus indicator that is white and blue. Then it can have sufficient contrast with both background colors at the same time.

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
